languagetool-language-modules/be/src/main/resources/org/languagetool/MessagesBundle_be.properties

```diff
@@ -4,30 +4,30 @@ atd = \u0412\u044b\u0437\u043d\u0430\u0447\u044b\u0446\u044c \u043c\u043e\u0432\
 ast = \u0410\u0441\u0442\u0443\u0440\u044b\u0439\u0441\u043a\u0430\u044f
```

Comment:
Please do not change this file in git, it needs to be edited on webtranslateit.com